### PR TITLE
docker type in the gap

### PR DESCRIPTION
**Pull request by nem035**
add question context for all docker questions and polish a few of the insights to be less noisy, more readable

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight

| Insight Name | Error Message |
|:-:|:-:|
| docker-cli | No matching footnotes found. |
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

File: docker/docker-fundamentals/docker-images/docker-image-registries.md [hunk 1]
```diff
@@ -57,7 +58,7 @@ Similarly, we can also add images to the registry by pushing them:
 docker push enki
 ```
 
-Alternatively, you can host your docker images on GCP, AWS, Azure, GitLab, JFrog or even on your own system.
+You're not required to host your images on Docker Hub and can setup your own registry as well.
```
Comment (nem035) [inline docker/docker-fundamentals/docker-images/docker-image-registries.md:61]:
```suggestion
You're not required to host your images on Docker Hub and can setup your own registry if you like.
```

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

File: docker/docker-fundamentals/docker-images/docker-image-tags.md [hunk 2]
```diff
@@ -10,17 +10,21 @@ links:
 
 # Image Tags
 
-
 ---
 
 ## Content
 
-A docker image tag is a similar concept to git tags. It is a pointer to a specific image commit (id) that serves to identify the image in a meaningful way.
+A docker image tag is a similar concept to [git tags](https://app.enkipro.com/insights/using-tags-for-version-control). It is a pointer to a specific image commit (id) that serves to identify the image in a meaningful way.
```
Comment (nem035) [inline docker/docker-fundamentals/docker-images/docker-image-tags.md:17]:
```suggestion
A docker image tag is a similar concept to [git tags](https://app.enki.com/insights/using-tags-for-version-control). It is a pointer to a specific image commit (id) that serves to identify the image in a meaningful way.
```

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |
### Insight

| Insight Name | Error Message |
|:-:|:-:|
| docker-engine | The practice question context `standalon` is unrecognized. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/docker/docker-fundamentals/containers/docker-container-vs-vm.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/docker/docker-fundamentals/containers/docker-container-vs-vm.md?ref=9d181e8a08b1dc26bc8cfcd7cb9e97c51fe139e2)
- [ ] [enkidevs/curriculum/docker/docker-fundamentals/containers/docker-containers.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/docker/docker-fundamentals/containers/docker-containers.md?ref=9d181e8a08b1dc26bc8cfcd7cb9e97c51fe139e2)
- [ ] [enkidevs/curriculum/docker/docker-fundamentals/containers/hardware-virtualization.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/docker/docker-fundamentals/containers/hardware-virtualization.md?ref=9d181e8a08b1dc26bc8cfcd7cb9e97c51fe139e2)
- [ ] [enkidevs/curriculum/docker/docker-fundamentals/docker-images/docker-image-registries.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/docker/docker-fundamentals/docker-images/docker-image-registries.md?ref=9d181e8a08b1dc26bc8cfcd7cb9e97c51fe139e2)
- [ ] [enkidevs/curriculum/docker/docker-fundamentals/docker-images/docker-image-tags.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/docker/docker-fundamentals/docker-images/docker-image-tags.md?ref=9d181e8a08b1dc26bc8cfcd7cb9e97c51fe139e2)
- [ ] [enkidevs/curriculum/docker/docker-fundamentals/docker-images/docker-images.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/docker/docker-fundamentals/docker-images/docker-images.md?ref=9d181e8a08b1dc26bc8cfcd7cb9e97c51fe139e2)
- [ ] [enkidevs/curriculum/docker/docker-fundamentals/docker-images/dockerfile-instructions.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/docker/docker-fundamentals/docker-images/dockerfile-instructions.md?ref=9d181e8a08b1dc26bc8cfcd7cb9e97c51fe139e2)
- [ ] [enkidevs/curriculum/docker/docker-fundamentals/networking/docker-dns.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/docker/docker-fundamentals/networking/docker-dns.md?ref=9d181e8a08b1dc26bc8cfcd7cb9e97c51fe139e2)
- [ ] [enkidevs/curriculum/docker/docker-fundamentals/networking/docker-networks.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/docker/docker-fundamentals/networking/docker-networks.md?ref=9d181e8a08b1dc26bc8cfcd7cb9e97c51fe139e2)
- [ ] [enkidevs/curriculum/docker/docker-fundamentals/networking/docker-port-forwarding.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/docker/docker-fundamentals/networking/docker-port-forwarding.md?ref=9d181e8a08b1dc26bc8cfcd7cb9e97c51fe139e2)
- [ ] [enkidevs/curriculum/docker/docker-fundamentals/overview/docker-cli.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/docker/docker-fundamentals/overview/docker-cli.md?ref=9d181e8a08b1dc26bc8cfcd7cb9e97c51fe139e2)
- [ ] [enkidevs/curriculum/docker/docker-fundamentals/overview/docker-engine.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/docker/docker-fundamentals/overview/docker-engine.md?ref=9d181e8a08b1dc26bc8cfcd7cb9e97c51fe139e2)
- [ ] [enkidevs/curriculum/docker/docker-fundamentals/overview/what-is-docker.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/docker/docker-fundamentals/overview/what-is-docker.md?ref=9d181e8a08b1dc26bc8cfcd7cb9e97c51fe139e2)
- [ ] [enkidevs/curriculum/docker/docker-fundamentals/storage/docker-bind-mounts.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/docker/docker-fundamentals/storage/docker-bind-mounts.md?ref=9d181e8a08b1dc26bc8cfcd7cb9e97c51fe139e2)
- [ ] [enkidevs/curriculum/docker/docker-fundamentals/storage/docker-intro.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/docker/docker-fundamentals/storage/docker-intro.md?ref=9d181e8a08b1dc26bc8cfcd7cb9e97c51fe139e2)
- [ ] [enkidevs/curriculum/docker/docker-fundamentals/storage/docker-tmpfs.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/docker/docker-fundamentals/storage/docker-tmpfs.md?ref=9d181e8a08b1dc26bc8cfcd7cb9e97c51fe139e2)
- [ ] [enkidevs/curriculum/docker/docker-fundamentals/storage/docker-volumes.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/docker/docker-fundamentals/storage/docker-volumes.md?ref=9d181e8a08b1dc26bc8cfcd7cb9e97c51fe139e2)

**Review by kapnobatai137 — APPROVED** [review]

**Pull request merged**